bolt-design-system/bolt
# Icon Update: Add solid star icon

## Jira

http://vjira2:8080/browse/BDS-858

## Summary

Adds a solid star icon to the Bolt icon library.

## Details

This adds the solid version of the star icon, since the design doesn't call for an outlined version, we are not adding that yet. This one will be named `star-solid` to go with our current naming conventions.

## How to test

Run the branch locally and navigate to the icons page. Make sure the star icon is there.
